### PR TITLE
da: Additional translations

### PR DESCRIPTION
**Pull request by thedataking**
We probably need to bikeshed whether we want to use "Flyttesemantik" or "Overførselssemantik" for "Move Semantics".

### CODE REVIEW

File: po/da.po [hunk 1]
```diff
@@ -160,7 +160,7 @@ msgstr "static & const"
 
 #: src/SUMMARY.md:46
 msgid "Scopes and Shadowing"
-msgstr ""
+msgstr "Omfang og overskyggede variabler"
```
Comment (mgeisler) [inline po/da.po:163]:
```suggestion
msgstr "Virkefelter og overskyggede variabler"
```

Jeg har relativt ofte hørt folk oversætte "scope" til ["virkefelt"](https://ordnet.dk/ddo/ordbog?entry_id=12007009&def_id=21095680&query=virkefelt).

**Comment by mgeisler** [comment]
> We probably need to bikeshed whether we want to use "Flyttesemantik" or "Overførselssemantik" for "Move Semantics".

Oh, good question! :smile: I see there is a Danish translation of the Rust Book (https://github.com/rust-lang/book/issues/3079, by @DanKHansen), but it is brand new and "move semantic" is not yet translated.

File: po/da.po (hunk 1 above)
Comment (mgeisler) [reply po/da.po:163]: Ah, jeg fandt et dokument som også bruger "virkefelt": https://homes.cs.aau.dk/~normark/c-prog-06/pdf/all.pdf.

**Review by mgeisler — APPROVED** [review]
I'm happy to get this in as a great base for further PRs!

File: po/da.po (hunk 1 above)
Comment (thedataking) [reply po/da.po:163]: Ah, jeg var på dybt vand mht. scope - virkefelt er en bedre oversættelse – tak! Today I learned.

**Pull request merged**